rosdep/python.yaml

```
@@ -8275,6 +8275,11 @@ python3-pyqt5.qtwebengine:
   openembedded: ['${PYTHON_PN}-pyqt5@meta-qt5']
   opensuse: [python3-qtwebengine-qt5]
   ubuntu: [python3-pyqt5.qtwebengine]
+python3-pyqt6.qtmultimedia:
+  debian: [python3-pyqt6.qtmultimedia]
```

[`Humble` and `Iron` run on `bullseye`](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027). The package only seems to have started to be available since `bookworm`: https://packages.debian.org/search?keywords=python3-pyqt6.qtmultimedia&searchon=names&suite=all&section=all

Maybe it's better to make sure it doesn't resolve to a nonexistent package in bullseye:

```suggestion
  debian:
    bullseye: null
    '*': [python3-pyqt6.qtmultimedia]
```

---

That makes sense; thanks for reviewing